cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -393,24 +451,38 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
             value: postgreServer.outputs.fqdn
           }
           {
-            name: 'POSTGRES_USERNAME'
-            value: databaseUser
+            name: 'POSTGRES_DATABASE'
+            value: postgreSqlDatabaseName
           }
           {
-            name: 'POSTGRES_DATABASE'
-            value: databaseName
+            name: 'POSTGRES_PORT'
+            value: '5432'
           }
           {
-            name: 'POSTGRES_PASSWORD'
-            secretRef: 'db-pass'
+            name: 'SPRING_DATASOURCE_URL'
+            value: 'jdbc:postgresql://${postgreServer.outputs.fqdn}:5432/${postgreSqlDatabaseName}'
           }
+          {{- end}}
+          {{- if (and .DbPostgres .DbPostgres.AuthUsingUsernamePassword) }}
           {
             name: 'POSTGRES_URL'
-            secretRef: 'db-url'
+            secretRef: 'postgresql-db-url'
```

Comment:
The `POSTGRES_URL` containers password. Keep the original implementation in this PR.